internal/pkg/constants/system.go

```diff
@@ -1,5 +1,8 @@
 package constants
 
+type UserIdCtxKey string
+type RoleCtxKey string
```

Comment:
don't define type for each ctx key, simply declare type ContextKey string and use it for all keys